app/scripts/lib/fxa-client.js

```diff
@@ -112,6 +112,16 @@ define(function (require, exports, module) {
           });
     },
 
+    checkAccountByEmailExists: function (email) {
```

Comment:
This name seems a bit strange, I had a hard time with a similar name over in #3419. Maybe `checkAccountEmailExists`?

How does `exists` work? Will it return `true` if the email is registered and not verified, or only if the email is registered and verified?